### PR TITLE
Es big num bug

### PR DESCRIPTION
**Pull request by lielShraga**

### CODE REVIEW

File: server/cmwell-it/src/it/scala/cmwell/it/ConsumeTests.scala [hunk 1]
```diff
@@ -126,6 +126,7 @@ class ConsumeTests extends AsyncFunSpec with Inspectors with Matchers with Helpe
           case t@(sta, pos, bod) => withClue(t) {
             sta should be(200)
             pos.isDefined should be(true)
+            println("consume body = " + bod)
```
Comment (bryaakov) [inline server/cmwell-it/src/it/scala/cmwell/it/ConsumeTests.scala]: Please remove `println` from tests

File: server/cmwell-it/src/it/scala/cmwell/it/ConsumeTests.scala [hunk 2]
```diff
@@ -249,6 +250,7 @@ class ConsumeTests extends AsyncFunSpec with Inspectors with Matchers with Helpe
           } {
             sta should be(200)
             pos.isDefined should be(true)
+            println("bod=" + bod)
```
Comment (bryaakov) [inline server/cmwell-it/src/it/scala/cmwell/it/ConsumeTests.scala]: same here

File: server/cmwell-it/src/it/scala/cmwell/it/ConsumeTests.scala (hunk 1 above)
Comment (lielShraga) [reply server/cmwell-it/src/it/scala/cmwell/it/ConsumeTests.scala]: done

File: server/cmwell-it/src/it/scala/cmwell/it/ConsumeTests.scala (hunk 2 above)
Comment (lielShraga) [reply server/cmwell-it/src/it/scala/cmwell/it/ConsumeTests.scala]: done

**Pull request merged**